api-reference/_hidden/GridBaseColumn/calculateFilterExpression.md

```diff
@@ -75,19 +77,20 @@ The default *"between"* implementation is inclusive (filter results include the
     import { Dx{WidgetName}Module } from "devextreme-angular";
     // ...
     export class AppComponent {
-        calculateFilterExpression (filterValue, selectedFilterOperation) {
-            const column = this as any;
+        calculateFilterExpression (this: Column, filterValue, selectedFilterOperation, target) {
```

Comment:
The Column object import is missing, see this demo as an example:
https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/CascadingLookups/Angular/Light/